src/components/new-request/search.vue

```diff
@@ -510,6 +512,15 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     return this.getEntityBlurbs?.find(type => type.value === entity_type_cd)?.blurbs
   }
 
+  get entity_type_options_select_bind (): EntityType | NrRequestTypeCodes {
+    if (this.isConversion) return this.getOriginEntityTypeCd
```

Comment:
> I got a rendering issue if binding `entity_type_cd` for alteration because it changed as we chose different alterations. Current solution for it is to change it to `origin_entity_type_cd` for alteration only.

So the issue is when we choose type of business alter into, the box cannot show value we select the first time we click on this value. But the value can be shown if we click on the same value twice.
![image](https://github.com/bcgov/namerequest/assets/60866283/3dcc66a1-9e71-4301-8ee1-a03b6734b4d8)

Reply:
Hmm, that needs to be fixed.

It sounds like a reactivity issue: something is not updating right away.

Reply:
Do you want to merge ASAP and then look into this one (same ticket, new PR)?

Reply:
Is it broken for other NR request  types? More specifically, what is broken right now?

Reply:
No, it works well now 😸

Reply:
OK, you can resolve this if you want.